eq_translations/extractable_strings.py

```diff
@@ -239,6 +239,71 @@
         "context": "Question",
         "additional_context": ["ListHeading", "ListDescription"],
     },
+    {
+        "json_path": "$..primary_content[*].title",
+        "description": "Introduction primary content",
+        "context": "PrimaryContent",
+    },
+    {
+        "json_path": "$..primary_content[*].contents[*].list[*]",
+        "description": "Introduction primary content list item",
+        "context": "PrimaryContent",
+        "additional_context": ["ListHeading", "ListDescription"],
+    },
+    {
+        "json_path": "$..primary_content[*].contents[*].description",
+        "description": "Introduction primary content description",
+        "context": "PrimaryContent",
+    },
+    {
+        "json_path": "$..primary_content[*].contents[*].guidance.contents[*].title",
+        "description": "Introduction primary content guidance title",
+        "context": "PrimaryContent",
+    },
+    {
+        "json_path": "$..primary_content[*].contents[*].guidance.contents[*].description",
+        "description": "Introduction primary content guidance description",
+        "context": "PrimaryContent",
+    },
+    {
+        "json_path": "$..preview_content.title",
+        "description": "Introduction preview content title",
+        "context": "PreviewContent",
+    },
+    {
+        "json_path": "$..preview_content.contents[*].description",
+        "description": "Introduction preview content description",
+        "context": "PreviewContent",
+    },
+    {
+        "json_path": "$..preview_content.questions[*].question",
+        "description": "Introduction preview content question title",
+        "context": "PreviewContent",
+    },
+    {
+        "json_path": "$..preview_content.questions[*].contents[*].description",
+        "description": "Introduction preview question content description",
+        "context": "PreviewContent",
+        "additional_context": ["PreviewQuestionListHeading"],
+    },
+    {
+        "json_path": "$..preview_content.questions[*].contents[*].list[*]",
+        "description": "Introduction preview question list item",
+        "context": "PreviewContent",
+        "additional_context": ["PreviewQuestionListHeading"],
+    },
+    {
+        "json_path": "$..secondary_content[*].contents[*].title",
+        "description": "Introduction secondary content contents title",
+    },
+    {
+        "json_path": "$..secondary_content[*].contents[*].list[*]",
+        "description": "Introduction secondary content contents list item",
+    },
+    {
+        "json_path": "$..secondary_content[*].contents[*].description",
+        "description": "Introduction secondary content contents description",
+    },
```

Comment:
Why is secondary not given the same level of context as primary?.

Reply:
It's because secondary is nested differently, title is part of contents, so I cannot create:
```
"SecondaryContent": {
        "parent_schema_property": "secondary_content",
        "property": "title",
        "text": "{context}",
    },
```
as there is no parent schema property for property "title" here.
Example:
```
"secondary_content": [
      {
          "id": "secondary-content",
          "contents": [
              {
                  "title": "Introduction additional title"
              },
              {
                  "description": "Introduction additional description"
              },
              {
                  "list": [
                      "Introduction additional list item",
                      "Introduction additional list item"
                  ]
              }
          ]
      }
  ]
```

Reply:
I don't think we can add `ListHeading` and `ListDescription`, secondary content is nested unlike anything else in schemas, title and description are separate dicts.

Reply:
Ok, I've added them on the same level so the items can now be tested with context, thanks!